sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -29,7 +29,7 @@ def __init__(self):
         self._client = None
 
     @classmethod
-    def load(cls, connection_string=None, endpoint=None, credential=None, **kwargs):
+    def load(cls, *, connection_string=None, endpoint=None, credential=None, **kwargs):
```

Comment:
You are making these arguments key word only? I am afraid they are breaking changes?

Reply:
I thought that as this was a beta then it was fine to make a breaking change?

Reply:
Yes. It is ok for beta to make breaking changes.

I just wanted to make sure they were by design.

Brw, do you want to update the doc

:param connection_string

to keyword connection_string?

Reply:
Missed that. Updated.

Reply:
I see you don't provide an async version of provider. I guess this is by design too?

Reply:
It was discussed originally. It can be something we add, but just don't have at the moment. Much like the SDK we would need to have a second version of provider to use the async client.

Do you think it is something we should add? It currently isn't high up on the backlog.

Reply:
If I remember correctly, we had shipped sync-only SDKs before, so it is probably ok.

And good to know you have an item to track it.